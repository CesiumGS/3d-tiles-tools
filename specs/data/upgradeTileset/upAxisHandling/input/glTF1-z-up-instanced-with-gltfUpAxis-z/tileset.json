{
  "asset": {
    "version": "1.0",
    "gltfUpAxis": "Z"
  },
  "geometricError": 4096,
  "root": {
    "boundingVolume": {
      "box": [3.0, 2.0, 0.5, 3.0, 0, 0, 0, 3.0, 0, 0, 0, 0.5]
    },
    "geometricError": 1024,
    "refine": "ADD",
    "content": {
      "uri": "Axes-glTF1-z-up.i3dm"
    }
  }
}
