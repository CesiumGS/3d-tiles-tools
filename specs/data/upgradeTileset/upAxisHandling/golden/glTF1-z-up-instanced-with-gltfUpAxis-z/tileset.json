{
  "asset": {
    "version": "1.1"
  },
  "geometricError": 4096,
  "root": {
    "boundingVolume": {
      "box": [
        3,
        2,
        0.5,
        3,
        0,
        0,
        0,
        3,
        0,
        0,
        0,
        0.5
      ]
    },
    "geometricError": 1024,
    "refine": "ADD",
    "content": {
      "uri": "Axes-glTF1-z-up.glb"
    }
  }
}